src/AccessibilityInsights.SharedUx/FileBug/BrowserEmulation.cs

```diff
@@ -0,0 +1,91 @@
+            SetBrowserFeatureControlKey("FEATURE_AJAX_CONNECTIONEVENTS", fileName, 1);
+            SetBrowserFeatureControlKey("FEATURE_GPU_RENDERING", fileName, 1);
+        }
+
+        private static void SetBrowserFeatureControlKey(string feature, string appName, uint value)
+        {
+            using (var key = Registry.CurrentUser.CreateSubKey(String.Concat("Software\\Microsoft\\Internet Explorer\\Main\\FeatureControl\\", feature), RegistryKeyPermissionCheck.ReadWriteSubTree))
+            {
+                key.SetValue(appName, (UInt32)value, RegistryValueKind.DWord);
+            }
+        }
+
+        private static UInt32 GetBrowserEmulationMode()
+        {
+            int browserVersion = 7;
+            using (var Key = Registry.LocalMachine.OpenSubKey("SOFTWARE\\Microsoft\\Internet Explorer", RegistryKeyPermissionCheck.ReadSubTree, System.Security.AccessControl.RegistryRights.QueryValues))
+            {
+                dynamic version = Key.GetValue("svcVersion");
+                if (null == version)
+                {
+                    version = Key.GetValue("Version");
+                    if (null == version)
+                    {
+                        throw new ApplicationException("Microsoft Internet Explorer is required!");
+                    }
+                }
+                int.TryParse(version.ToString().Split('.')[0], out browserVersion);
+            }
+
+            UInt32 mode = 10000;
+            // Internet Explorer 10. Webpages containing standards-based !DOCTYPE directives are displayed in IE10 Standards mode. Default value for Internet Explorer 10.
+            switch (browserVersion)
+            {
+                case 7:
+                    mode = 7000;
+                    // Webpages containing standards-based !DOCTYPE directives are displayed in IE7 Standards mode. Default value for applications hosting the WebBrowser Control.
+                    break;
+                case 8:
+                    mode = 8000;
+                    // Webpages containing standards-based !DOCTYPE directives are displayed in IE8 mode. Default value for Internet Explorer 8
+                    break;
+                case 9:
+                    mode = 9000;
+                    // Internet Explorer 9. Webpages containing standards-based !DOCTYPE directives are displayed in IE9 mode. Default value for Internet Explorer 9.
+                    break;
+                case 10:
+                    mode = 10000;
+                    // Internet Explorer 10. Webpages containing standards-based !DOCTYPE directives are displayed in IE9 mode. Default value for Internet Explorer 9.
+                    break;
+                case 11:
+                    mode = 11001;
+                    // Internet Explorer 11. Webpages containing standards-based !DOCTYPE directives are displayed in IE9 mode. Default value for Internet Explorer 9.
+                    break;
+            }
+
+            return mode;
```

Comment:
In a switch statement like this, it's sometimes clearer with a more direct approach, such as
```suggestion
                    case 7: return 7000;
                    case 8: return 8000; 
                    case 9: return 9000;
                    case 11: return 11001
                    case 10:
                    default:
                        return 10000;
```